### ISSUE TITLE
[Feature request] Change toast notification width based on song name length

### ISSUE THREAD

**Issue by cs127** [Scotsguy/now-playing#7]
A lot of custom songs in my resource pack have long names and I would like this mod to show wider notifications to have the entire text visible.
I don't actually know if resizing the toasts is easily possible to do though. If it is possible, I would really appreciate it if you added this feature! Also I use Fabric.

**Comment by Scotsguy** [comment]
Sounds reasonably doable, would just need to repeat the middle part of the texture until the name fits. Do you have a resource pack I could test this with?

**Comment by cs127** [comment]
Sorry for the late response! I've been busy and I forgot about this!
Here's a test resource pack that I made right now, it replaces the `music.game` sound event with only two of the Minecraft songs, and I added a custom name for both of them in the `assets/now_playing/lang/en_us.json` language file.

[Here's the resource pack.](https://github.com/Scotsguy/now-playing/files/6421879/test_pack.zip)

This is how it looks in the game:
![screenshot](https://user-images.githubusercontent.com/56088587/117021702-d1712c80-ad0c-11eb-8cf0-439e95346011.png)
